doc/usecases/record_elektra/UC_ansible-libelektra_marked_keys.md

```diff
@@ -0,0 +1,18 @@
+# Use Case: Fail application of ansible-libelektra if certain keys differ in value
+
+## Summary
+
+- **Title:** Fail application of ansible-libelektra if certain keys differ in value
```

Comment:
Useless to state the same title again.

Reply:
But the template clearly states so...

![grafik](https://user-images.githubusercontent.com/5960633/189962092-5dcc11bb-83fc-40d0-9808-2b10da645fc3.png)

Reply:
As you want :wink: